scripts/populate_mock_db.py

```
@@ -0,0 +1,69 @@
+    c.execute(
+        "CREATE TABLE appliance (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT NOT NULL UNIQUE)"
+    )
+    c.execute(
+        "CREATE TABLE appliance_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, appliance_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+
+    # Add 50 crew members
+    roles = ["WC", "CM", "CC", "FFC", "FFD", "FFT"]
+    skills_list = ["BA", "BA LGV", "BA TTR", "LGV", "TTR"]
+
+    now = datetime.now()
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)